buildSrc/src/main/kotlin/plugin/AllProjectConfigurer.kt

```diff
@@ -184,8 +184,7 @@ class AllProjectConfigurer {
       implementation(Libs.collection_ktx)
       implementation(Libs.butterknife)
       kapt(Libs.butterknife_compiler)
-      implementation(Libs.xfetch2)
-      implementation(Libs.xfetch2okhttp)
```

Comment:
We need to `fetch2okhttp` for testing the download. How we are gonna implemented this? As the author didn't publish this module on Jitpack.

Reply:
yes the library on ```jitpack``` contain both ```xfetch2``` and ```xfetch2okhttp``` in one implementation.